metacat-elasticsearch/src/main/java/com/netflix/metacat/elasticsearch/search/ElasticSearchDoc.java

```diff
@@ -59,7 +57,7 @@
     private boolean deleted;
     private String refreshMarker;
     private boolean addSearchableDefinitionMetadataEnabled = true;
-
+    private MetacatJson metacatJson = new MetacatJsonLocator();
```

Comment:
Hard to see from the context but I don't understand why  this is sent into the document file itself instead of being external as a serializer for this document class?